### PR TITLE
julienschmidt/httprouter: support extra span options in httprouter

### LINKED ISSUE #345: julienschmidt/httprouter: support additional span options for httprouter
It would be nice if the httprouter integration allowed for specifying additional span options similar to the gorilla/mux integration.

https://github.com/DataDog/dd-trace-go/blob/v1.4.0/contrib/gorilla/mux/option.go#L26

### PR DESCRIPTION
**Pull request by phette**
The httprouter integration did not allow for providing additional span options. I have applied the exact same logic from the gorilla/mux integration.

Closes #345

### CODE REVIEW

**Comment by phette** [comment]
Unfortunately the linter is failing in a different contrib than I touched.  Happy to fix it as a part of this PR if preferred.

**Comment by gbbr** [comment]
Seems legit, and yes, we can not merge anything that isn't green, so please address that if you can. It might be that the `mongo` dependency changed.

File: contrib/julienschmidt/httprouter/option.go [hunk 1]
```diff
@@ -15,3 +20,10 @@ func WithServiceName(name string) RouterOption {
 		cfg.serviceName = name
 	}
 }
+
+// WithSpanOptions applies the given set of options to the spans started by the router.
```
Comment (gbbr) [inline contrib/julienschmidt/httprouter/option.go]: Can we make this singular? s/spans/span

**Comment by gbbr** [comment]
Fixed CI. Just a minor change left (my comment) and am happy to merge. Will include this in the next release. Thanks!

File: contrib/julienschmidt/httprouter/option.go (hunk 1 above)
Comment (phette) [reply contrib/julienschmidt/httprouter/option.go]: This takes in a slice of span options so technically singular isn't correct as I would like the ability to add multiple span options when initializing the router.

[the remark this answers is not in the document]
Comment (phette) [reply contrib/julienschmidt/httprouter/option.go]: Ahh yes my bad.

**Pull request merged**